### PR TITLE
Remove NormalizationForm from System.Globalization.Extensions

### PR DESCRIPTION
**Pull request by tarekgh**
NormalizationForm is now exposed from System.Runtime so we'll have to remove it from System.Globalization.Extensions contratc and now the library System.Globalization.Extensions becomes a patial facade as now we type-forward NormalizationForm

### CODE REVIEW

**Comment by tarekgh** [comment]
#11747

**Comment by tarekgh** [comment]
@weshaggard PTAL. thanks.

File: src/System.Globalization.Extensions/tests/System.Globalization.Extensions.Tests.csproj [hunk 1]
```diff
@@ -11,7 +11,8 @@
     <RootNamespace>System.Globalization.Extensions.Tests</RootNamespace>
     <AssemblyName>System.Globalization.Extensions.Tests</AssemblyName>
     <ProjectGuid>{BC439554-4AB4-4C94-8E28-C00EDE4FD1C7}</ProjectGuid>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker Condition="'$TargetGroup)' == ''">.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker Condition="'$TargetGroup)' == 'netstandard1.7'">.NETStandard,Version=v1.7</NugetTargetMoniker>
```
Comment (weshaggard) [inline src/System.Globalization.Extensions/tests/System.Globalization.Extensions.Tests.csproj]: This shouldn't be needed as it gets set in dir.props.

File: src/System.Globalization.Extensions/tests/System.Globalization.Extensions.Tests.csproj [hunk 2]
```diff
@@ -11,7 +11,8 @@
     <RootNamespace>System.Globalization.Extensions.Tests</RootNamespace>
     <AssemblyName>System.Globalization.Extensions.Tests</AssemblyName>
     <ProjectGuid>{BC439554-4AB4-4C94-8E28-C00EDE4FD1C7}</ProjectGuid>
-    <NugetTargetMoniker>.NETStandard,Version=v1.3</NugetTargetMoniker>
+    <NugetTargetMoniker Condition="'$TargetGroup)' == ''">.NETStandard,Version=v1.3</NugetTargetMoniker>
```
Comment (weshaggard) [inline src/System.Globalization.Extensions/tests/System.Globalization.Extensions.Tests.csproj]: `'$TargetGroup)` -> `'$(TargetGroup)'`

**Comment by tarekgh** [comment]
@weshaggard I have fixed it according to your comments. thanks for your review.

**Pull request merged**